domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/behandling/KanIkkeHenteBehandling.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.tiltakspenger.saksbehandling.domene.behandling
+
+sealed interface KanIkkeHenteBehandling {
+    data object MåVæreSaksbehandlerEllerBeslutter : KanIkkeHenteBehandling
```

Comment:
I en fremtidig kotlin-versjon kommer det union types for Errors. Da kan vi endre til å gjenbruke mange av disse generelle feiltypene